### PR TITLE
Add skeleton to dev

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 

-   [x] Have you added an explanation of what your changes do and why you'd like them to be included?
-   [x] Have you updated or added documentation for the change?
-   [x] Have you tested your changes with successful results?

**Type of Changes:**

 

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Documentation (non-breaking change which adds documentation)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

- Currently there is no react app. #1

**What is the new behavior (if this is a feature change)?**

- Now there is an react app. #1 

**Other information**:

- Ref: https://decembersoft.com/posts/starting-a-react-redux-project-with-typescript/

### CODE REVIEW

File: src/components/pages/HomePage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * Provides its connected component with the pieces of the data it needs from
+ * the store, and the functions it can use to dispatch actions to the store.
+ */
+export default HomePage;
+//#endregion
+
+/**
+ * SPDX-License-Identifier: (EUPL-1.2)
+ * Copyright © 2019-2020 Simon Prast
```
Comment (UnknownExecute) [inline src/components/pages/HomePage/index.tsx]: Change Simon Prast to Nico Schett

**Review by UnknownExecute — APPROVED** [review]
Looks good to merge

File: .gitignore [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
+.idea
 
-# Fonts
-*.ttf
+# Logs
+*.log*
 
 # SPDX-License-Identifier: (EUPL-1.2)
-# Copyright © 2019-2020 Simon Prast
+# Copyright © Nico Schett
```
Comment (schettn) [inline .gitignore]: newline missing
```suggestion
# Copyright © Nico Schett

```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "front",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
```
Comment (schettn) [inline package.json]: Those should be moved to the dev dependencies.

File: src/App.test.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```
Comment (schettn) [inline src/App.test.tsx]: The test case is no longer valid. We should remove it for now.

File: src/components/atoms/ScrollToTop/index.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  render() {
+    return this.props.children;
+  }
+}
+//#endregion
+
+//#region > Exports
+//> Default Component
```
Comment (schettn) [inline src/components/atoms/ScrollToTop/index.ts]:
```suggestion
```
 This is fucking unnecessary. I suggest to remove it form **all** files.

**Review by schettn — CHANGES_REQUESTED** [review]
Looks good after few changes. Furthermore have a look at all legacy files like `src/logo.svg.

**Review by schettn — APPROVED** [review]
lgtm

**Pull request merged**